packages/plugin-stdio/src/StdioPlugin.ts

```
@@ -52,7 +52,7 @@ export class StdioPlugin implements IPlugin, ITransport {
     this.commandHandlers.forEach((handler, commandKey) => {
         // Assuming commandKey is "command" or "command sub"
         const [cmd, ...subcommands] = commandKey.split(' ');
-        const commandModule: CommandModule = {
+        const commandModule: CommandModule<{}, {}> = {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)